lib/vanagon/driver.rb

```diff
@@ -76,14 +76,13 @@ def run
       @engine.startup(@workdir)
 
       puts "Target is #{@engine.target}"
-
-      install_build_dependencies
+      retry_task { install_build_dependencies }
       @project.fetch_sources(@workdir)
       @project.make_makefile(@workdir)
       @project.make_bill_of_materials(@workdir)
       @project.generate_packaging_artifacts(@workdir)
       @engine.ship_workdir(@workdir)
-      @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make})")
+      retry_task { @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make})") }
```

Comment:
I was thinking about this; should we assume that all engines get this part retried or should it happen per-engine? I'm open to trying it this way; I was just thinking about interface standardization vs. RE convenience earlier today.